src/loc/pt-br.ts

```diff
@@ -0,0 +1,358 @@
+      "L_RelativeDateTime_Tomorrow": "Amanhã",
+      "L_RelativeDateTime_Yesterday": "Ontem",
+      "L_RelativeDateTime_TomorrowAndTime": "Amanhã às {0}",
+      "L_RelativeDateTime_YesterdayAndTime": "Ontem às {0}",
+      "L_RelativeDateTime_XHoursFuture": "Em {0} hora || Em {0} horas",
+      "L_RelativeDateTime_XHours": "{0} hora atrás || {0} horas atrás",
+      "L_RelativeDateTime_XHoursFutureIntervals": "1 || 2-",
+      "L_RelativeDateTime_XHoursIntervals": "1 || 2-",
+      "L_RelativeDateTime_DayAndTime": "{0} em {1}",
+      "L_RelativeDateTime_XDaysFuture": "{0} dias a partir de agora || {0} dias a partir de agora",
+      "L_RelativeDateTime_XDays": "{0} dia atrás || {0} dias atrás",
+      "L_RelativeDateTime_XDaysFutureIntervals": "1 || 2-",
+      "L_RelativeDateTime_XDaysIntervals": "1 || 2-",
+      "L_RelativeDateTime_Today": "Hoje"
+    },
+    "SendEmailTo": "Envie um e-mail para {0}",
+    "StartChatWith": "Inicie um bate-papo com {0}",
+    "Contact": "Contato",
+    "UpdateProfile": "Atualize seu perfil",
+
+    "TaxonomyPickerNoTerms": "O conjunto de termos não contém quaisquer termos",
+    "TaxonomyPickerExpandTitle": "Expandir este conjunto de termos",
+    "TaxonomyPickerMenuTermSet": "Menu para conjunto de termos",
+    "TaxonomyPickerInLabel": "no",
+    "TaxonomyPickerTermSetLabel": "Conjunto de termos",
+    "TaxonomyPickerTermsNotFound": "Os próximos termos selecionados não foram encontrados no repositório de termos: {0}",
+    "TaxonomyPickerInvalidTerms": "Corrija os termos inválidos: {0}",
+
+    peoplePickerComponentTooltipMessage: "Selecionador de Pessoas",
+    peoplePickerComponentErrorMessage: "Campo obrigatório",
+    peoplePickerSuggestionsHeaderText: 'Pessoas sugeridas',
+    peoplePickerLoadingText: 'Carregando',
+    PeoplePickerSearchText: 'Buscando usuários',
+    PeoplePickerGroupNotFound: "Grupo não encontrado.",
+
+    genericNoResultsFoundText: 'Nenhum resultado encontrado',
+
+    ListItemPickerSelectValue: 'Selecione o valor',
+    ListItemAttachmentsActionDeleteIconTitle: 'Excluir',
+    ListItemAttachmentsactionDeleteTitle: 'Excluir',
+    ListItemAttachmentsfileDeletedMsg: 'Arquivo {0} excluído',
+    ListItemAttachmentsfileDeleteError: 'Erro ao excluir arquivo: {0}, motivo {1}',
+    ListItemAttachmentserrorLoadAttachments: 'Erro ao carregar o anexo do item da lista, motivo: {0}',
+    ListItemAttachmentsconfirmDelete: 'Tem certeza de que deseja enviar o anexo {0} para a lixeira do site?',
+    ListItemAttachmentsdialogTitle: 'Anexo de item de lista',
+    ListItemAttachmentsdialogOKbuttonLabel: 'OK',
+    ListItemAttachmentsdialogCancelButtonLabel: 'Cancelar',
+    ListItemAttachmentsdialogOKbuttonLabelOnDelete: 'Excluir',
+    ListItemAttachmentsuploadAttachmentDialogTitle: 'Juntar anexo',
+    ListItemAttachmentsuploadAttachmentButtonLabel: 'Juntar anexo',
+    ListItemAttachmentsuploadAttachmentErrorMsg: 'O arquivo {0} não anexado, motivo: {1}',
```

Comment:
ListItemAttachmentsuploadAttachmentErrorMsg: 'O arquivo {0} não anexado, motivo: {1}', should be 
 ListItemAttachmentsuploadAttachmentErrorMsg: 'O arquivo {0} não **foi** anexado, motivo: {1}',